torch/csrc/jit/passes/quantization.cpp

```diff
@@ -203,7 +228,9 @@ void InsertObserverNodes(const std::unique_ptr<script::Method>& method,
   }
 }
 
-void InsertQuantDequantNodes(std::shared_ptr<Graph>& graph) {
+void InsertQuantDequantNodes(std::shared_ptr<Graph>& graph,
+  const std::unordered_map<std::string, std::tuple<std::string,
```

Comment:
Maybe create a struct instead of the tuple? It's hard to tell what fields is what.